client/dashboard/app/queries.ts

```diff
@@ -163,6 +171,31 @@ export function restoreSitePlanSoftwareMutation( siteId: string ) {
 	};
 }
 
+export function siteOwnerTransferMutation( siteId: string ) {
+	return {
+		mutationFn: ( newData: { new_site_owner: string } ) => siteOwnerTransfer( siteId, newData ),
+	};
+}
+
+export function siteOwnerTransferEligibilityCheckMutation( siteId: string ) {
+	return {
+		mutationFn: ( newData: { new_site_owner: string } ) =>
+			siteOwnerTransferEligibilityCheck( siteId, newData ),
+	};
+}
+
+export function siteOwnerTransferConfirmMutation( siteId: string ) {
+	return {
+		mutationFn: ( newData: { hash: string } ) => siteOwnerTransferConfirm( siteId, newData ),
```

Comment:
Super minor, but `newData` is probably a misnomer since we're passing an existing hash to the server to execute the transfer.

You could consider swapping `newData` in these functions to clarify the intentions of the backend. Something like:
- `transferData`
- `eligibilityData`
- `confirmationData`

Reply:
Or just `data` 🙈